pandas/tests/series/methods/test_shift.py

```diff
@@ -220,10 +223,58 @@ def test_tshift(self, datetime_series):
         tm.assert_series_equal(unshifted, inferred_ts)
 
         no_freq = datetime_series[[0, 5, 7]]
-        msg = "Freq was not given and was not set in the index"
+        msg = "Freq was not set in the index hence cannot be inferred"
         with pytest.raises(ValueError, match=msg):
             no_freq.tshift()
 
+    def test_tshift_deprecated(self, datetime_series):
+        # GH#11631
+        with tm.assert_produces_warning(FutureWarning):
+            datetime_series.tshift()
+
+    def test_shift_with_freq(self, datetime_series):
```

Comment:
same comment as above, e.g. to split this up a bit

Reply:
done